src/transform.ts

```diff
@@ -19,7 +19,7 @@ export async function transform(
 ) {
   id = toPosixPath(id)
   root = toPosixPath(root)
-  const dir = dirname(id)
+  const dir = !id.includes('/') ? null : dirname(id)
```

Comment:
This might not be a safe assumption to me, a version module could still have `/` like `virtual:some/thing`.

How does Vite core handle that?

Reply:
👍 Done https://github.com/antfu/vite-plugin-glob/pull/18/commits/1d4202e50423cc9d8463fb67693dc732b209f831.